package.json

```
@@ -49,6 +49,7 @@
         "jest-fetch-mock": "3.0.3",
         "lint-staged": "15.2.10",
         "prettier": "3.3.3",
+        "rimraf": "^5.0.10",
```

All other dependencies version are pinned
```suggestion
        "rimraf": "5.0.10",
```